cdap-support-bundle/bin/observability.py

```diff
@@ -0,0 +1,539 @@
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+# observability.py to get logs from CDAP instance deployed on kubernetes.
+
+import argparse
+import base64
+import json
+from json.decoder import JSONDecodeError
+import os
+import shutil
+import subprocess
+import sys
+from datetime import datetime
+
+parser = argparse.ArgumentParser(description='Log collector program for CDAP.')
+parser.add_argument(
+    "--namespace",
+    "-n",
+    help="Kubernetes namespace where CDAP is installed",
+    required=True,
+    type=str
+)
+parser.add_argument(
+    "--instance-name",
+    "-in",
```

Comment:
should be single character after -.
i.e. -i instead of -in. Similar comment for other params

Reply:
We have removed the instance-name from parameters. Changed the alias for other params to single character.